code/src/components/Hover.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Hover = (props) => {
```

Comment:
Maybe just a matter of taste but personally think it would be easier to follow if naming it Icons or IconHover

Reply:
Also seems like the props is not used in this component :)